migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```

Comment:
unused?

Reply:
it was generated automatically, I'll remove it

Reply:
it was generated automatically, I'll remove it